draft-ietf-quic-transport.md

```diff
@@ -4697,11 +4697,11 @@ active_connection_id_limit (0x000e):
   to store. This value includes the connection ID received during the handshake,
   that received in the preferred_address transport parameter, and those received
   in NEW_CONNECTION_ID frames.
-  Unless a zero-length connection ID is being used, the value of the
+  Unless the peer is using a zero-length connection ID, the value of the
   active_connection_id_limit parameter MUST be no less than 2. If this
-  transport parameter is absent, a default of 2 is assumed.
-  When a zero-length connection ID is being used, the active_connection_id_limit
-  parameter MUST NOT be sent.
+  transport parameter is absent, a default of 2 is assumed.  When the client
+  is using a zero-length connection ID, the server MUST NOT send the
+  active_connection_id_limit parameter.
```

Comment:
I would say here that the server MAY omit the parameter.  Sending a fixed set of transport parameters is a good practice.

Reply:
I was trying to keep this an editorial PR - saying that the server MAY omit the parameter changes normative language, which I thought would make it design.

Reply:
The MUST NOT is gone, leaving an implicit MAY.